terraform/aks/app.tf

```diff
@@ -224,7 +225,7 @@ module "worker_application_configuration" {
 }
 
 module "worker_application" {
-  source     = "git::https://github.com/DFE-Digital/terraform-modules.git//aks/application?ref=testing"
+  source     = "git::https://github.com/DFE-Digital/terraform-modules.git//aks/application?ref=stable"
```

Comment:
Not sure we should just change these 2 apps to point to stable, while everything else is testing. There is a card to migrate TRS to terrafile and it will be picked up then.